samples/apps/copilot-chat-app/webapi/PlannerSkills/Summarize/skprompt.txt

```diff
@@ -0,0 +1,23 @@
+[SUMMARIZATION RULES]
+DONT WASTE WORDS
```

Comment:
Just curious... Does using ALL CAPS make a difference for the prompt?

Reply:
All caps definitely makes a difference with how the LLM interprets the directions. This raises the importance of the directions in this prompt.